src/core/Table/TablePaginator.tsx

```diff
@@ -311,6 +325,11 @@ export const TablePaginator = (props: TablePaginatorProps) => {
         </IconButton>
       </div>
       <div className='iui-right'>
+        {localization.rowsPerPageLabel && paginatorWidth >= 1024 && (
```

Comment:
Label shouldn't be shown if pageSizeList is undefined

Reply:
Fixed by merging main